src/components/Book.js

```diff
@@ -3,25 +3,51 @@ import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/apiReducer';
 
 function Book({ id, title, author }) {
+  const progress = Math.floor(Math.random() * 101);
+  const chapter = Math.floor(Math.random() * 10);
   const dispatch = useDispatch();
   const handleClick = () => {
-    console.log(id);
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h2>
-        Title :
-        {' '}
-        {title}
-      </h2>
-      <h4>
-        Author :
-        {' '}
-        {author}
-        {' '}
-      </h4>
-      <button type="button" onClick={() => handleClick()}>Remove</button>
+    <div className="BOOK_CONTAINER">
```

Comment:
- [ ] The book list should be centered on the page, also kindly add the profile icon in the header so your design looks like the Zeplin one

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220729_064842](https://user-images.githubusercontent.com/25227022/181692138-bf3ba1d4-eef3-4c30-a58e-cf0aafc45973.png)|![Screenshot_20220728_202309](https://user-images.githubusercontent.com/25227022/181692139-4bb89505-4e20-4a42-99d6-b3828e715550.png)|